### PR TITLE
Initial commit for Hamburger Navbar

### PR DESCRIPTION
**Pull request by Ankitasingh88**
Added the first draft of Hamburger Nav Menu and Footer

### CODE REVIEW

File: styles/layout.css [hunk 1]
```diff
@@ -46,16 +46,52 @@ body{
...
     color: white;
     font-size: 1.5rem;
+    text-decoration: none;
+}
+
+.nav-link{
+    transition: 0.7s ease;
+}
+
+.nav-link:hover{
```
Comment (VaishaliGovindaraj) [inline styles/layout.css]: Can we make an indicator to  highlight the current page ankita .. 
Example: if we are in mammals , see the attachemnt below:

Active_page_indication

something like this.

Comment (Ankitasingh88) [reply styles/layout.css]: Code changes to make the navbar active have been pushed to this pull request. Have a look

**Review by Lvhacker1 — DISMISSED** [review]
my task was to make the header and footer. you can remove/edit the code and send a new pr.

**Comment by Ankitasingh88** [comment]
> my task was to make the header and footer. you can remove/edit the code and send a new pr.

You can build your changes on top of my changes

**Comment by PriyeshIllam** [comment]
LGTM

**Comment by Lvhacker1** [comment]
You need to merge the changes into the development branch and ensure that they are tested before merging them into the main branch.

**Comment by VaishaliGovindaraj** [comment]
hamburger menu is looking good

**Review by PriyeshIllam — APPROVED** [review]
LGTM

**Review by VaishaliGovindaraj — APPROVED** [review]

**Pull request merged**